x-pack/legacy/plugins/uptime/public/components/functional/snapshot.tsx

```diff
@@ -23,74 +25,34 @@ interface SnapshotQueryResult {
  * glean the status of their uptime environment.
  * @param props the props required by the component
  */
-export const SnapshotComponent = ({ data }: UptimeGraphQLQueryProps<SnapshotQueryResult>) =>
-  data && data.snapshot ? (
-    <React.Fragment>
-      <EuiPanel>
-        <EuiTitle size="xs">
-          <h5>
-            <FormattedMessage
-              id="xpack.uptime.snapshot.endpointStatusTitle"
-              defaultMessage="Current status"
-            />
-          </h5>
-        </EuiTitle>
-        <EuiFlexGroup direction="column" gutterSize="m">
-          <EuiFlexItem grow={false}>
-            <EuiSpacer size="xs" />
-          </EuiFlexItem>
-          <EuiFlexItem>
-            <EuiFlexGroup justifyContent="spaceEvenly" gutterSize="s">
-              <EuiFlexItem>
-                <EuiStat
-                  description={i18n.translate('xpack.uptime.snapshot.stats.upDescription', {
-                    defaultMessage: 'Up',
-                  })}
-                  textAlign="center"
-                  title={data.snapshot.counts.up}
-                  titleColor="secondary"
-                />
-              </EuiFlexItem>
-              <EuiFlexItem>
-                <EuiStat
-                  description={i18n.translate('xpack.uptime.snapshot.stats.downDescription', {
-                    defaultMessage: 'Down',
-                  })}
-                  textAlign="center"
-                  title={data.snapshot.counts.down}
-                  titleColor="danger"
-                />
-              </EuiFlexItem>
-              {data.snapshot.counts.mixed > 0 ? (
-                <EuiFlexItem>
-                  <EuiStat
-                    description={i18n.translate('xpack.uptime.snapshot.stats.mixedDescription', {
-                      defaultMessage: 'Mixed',
-                    })}
-                    textAlign="center"
-                    title={data.snapshot.counts.mixed}
-                    titleColor="subdued"
-                  />
-                </EuiFlexItem>
-              ) : null}
-              <EuiFlexItem>
-                <EuiStat
-                  description={i18n.translate('xpack.uptime.snapshot.stats.totalDescription', {
-                    defaultMessage: 'Total',
-                  })}
-                  textAlign="center"
-                  title={data.snapshot.counts.total}
-                  titleColor="subdued"
-                />
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          </EuiFlexItem>
-        </EuiFlexGroup>
-      </EuiPanel>
-    </React.Fragment>
-  ) : (
-    <SnapshotLoading />
-  );
+export const SnapshotComponent = ({
+  data,
+  loading,
+}: UptimeGraphQLQueryProps<SnapshotQueryResult>) => (
+  <ChartWrapper loading={loading}>
+    <EuiTitle size="xs">
+      <h5>
```

Comment:
I suggest using a `<h2>` element.

Reply:
Fixed in https://github.com/elastic/kibana/pull/47176/commits/8c65f72f90a1af07381c4a7e05044958f4c22214.